agent/util-scripts/unittests

```diff
@@ -63,6 +66,12 @@ let res=res+${?}
 cp -rL ${_tdir}/test-bin/* ${_testopt}/unittest-scripts/
 let res=res+${?}
 
+export _testhome=${_testroot}/home
+mkdir ${_testhome}
+let res=res+${?}
+ln -s ${HOME}/.local ${_testhome}/.local
```

Comment:
I believe this is worth moving forward with by filing an issue to remove it later, accepting this PR as is, moving on to focus on PR #2649, so that we can gain the benefits of the new `tox` "envs" and the parallelism while we work on solving the test infrastructure independently.

Reply:
FWIW, the run time of tests in the CI under PR #2649 is roughly 8 minutes, while the run time under this PR is 32 minutes (roughly).  The run time of `main` CI jobs is 29 minutes.

Reply:
Peter has assured me that this change limits and contains the Badness better than his previous change, that he's going to open an issue to note the existence of this Badness, and that it will soon be removed as we continue in our evolution toward better use of Tox.

Reply:
Filed tracking issue #2680.